.github/workflows/preview.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish Preview release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish:
+    name: Publish Preview to VS Code Marketplace
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Set preview in package.json
+        id: set-preview
+        run: |
+          ./build/preview
+
+      - name: Read extension version
+        id: ext-version
+        run: |
+          content=`cat ./package.json | jq -r .version`
+          echo "::set-output name=content::$content"
+
+      - name: Read Node.js version
+        id: nodejs-version
+        run: |
+          content=`cat ./.nvmrc`
+          echo "::set-output name=content::$content"
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
```

Comment:
Would it be possible to use `v2` here?

Reply:
Good point. This was a copypasta from our https://github.com/hashicorp/vscode-terraform/blob/main/.github/workflows/publish.yml#L33 so we should probably update that there too